### PR TITLE
Npm workspaces

### PR DESCRIPTION
**Pull request by TheEvilDev**
## Overview
Removes dev dependencies on `yarn` and `lerna` and just uses npm workspaces.

## Tests
[Include a link to the passing GitHub action running the test suite here.]

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -3,18 +3,44 @@
...
+```
+
+4. Provide credentials
+   Copy a set of AWS JSON formatted credentials to the account with the access to the SiteWise resources you are requesting, at `<rootDir>/packages/components/creds.json`.
+
+These credentials should not be uploaded to the repository.
+
+5. Point to valid SiteWise resources
+
+Alter the file `siteWiseQueries` within `@iot-app-kit/components` to point to valid SiteWise asset properties for the accounts credentials from the prior step.
```
Comment (boweihan) [inline docs/development.md:43]: dunno if it's just GH, but the spacing of these descriptions look inconsistent (new line here, vs. tab above and below).

File: docs/development.md [hunk 2]
```diff
@@ -3,18 +3,44 @@
 ### Environment setup
 
 Iot-App-Kit runs on node and uses yarn for package management. Specific versions are required in order to keep builds working with the current monorepo tooling.
```
Comment (boweihan) [inline docs/development.md:5]: Remove yarn from this statement

**Review by boweihan — CHANGES_REQUESTED** [review]
There are a few instances of documentation that still use yarn commands - for example, `docs/GettingStarted.md`, `README.md`, and our `npm-publish.yml` workflow. Could probably just do a global find and replace for these instances.

**Comment by diehbria** [comment]
make sure to clean up the commit history (which is considered part of the review)

**Comment by boweihan** [comment]
Closing this and rolling it into https://github.com/awslabs/iot-app-kit/pull/231

**Pull request closed without merging**